packages/components/src/components/permissions/models.ts

```diff
@@ -223,6 +226,10 @@ export class SecurityPolicy extends Record({
             assignmentsByRole: SecurityPolicy.getAssignmentsByRole(assignments)
         }) as SecurityPolicy;
     }
+
+    isInheritFromParent(): boolean {
+        return this.containerId && this.resourceId !== this.containerId;
```

Comment:
Any chance that resourceId is undefined here?  I'm not sure what that would mean...

Reply:
No. I don't think that it can be undefined here (at least not for a real LK scenario...I suppose it could be in a jest test). Even if it is, I think having undefined in that conditional should be fine.